OracleWebLogic/samples/12212-domain/container-scripts/domain.properties

```diff
@@ -0,0 +1,10 @@
+DOMAIN_NAME=abc_domain
+ADMIN_PORT=7001
+ADMIN_HOST=wlsadmin
+CLUSTER_NAME=DockerCluster
+PRODUCTION_MODE=dev
+ADMIN_PASSWORD=welcome4
```

Comment:
We can't accept any hard-coded passwords. Can you remove this and manually set it on container start somehow? Though, I may be misunderstanding how this interacts with `createServer.sh`.